MediaAdminBundle/Resources/public/coffee/orchestraAbstractMediaType.coffee

```diff
@@ -0,0 +1,59 @@
+extendView = extendView || {}
+extendView['orchestraMediaAbstractType'] =
+  events:
+    'click .clear-media': 'clearMedia'
```

Comment:
If you have no clear-media in tinyMce, this event should'nt be in this extendView.